BugsAndExploits/TntDuping.md

```diff
@@ -6,15 +7,21 @@ description: Explaination of tnt duplication
 # tnt duplication
 
 ## History
-TNT Duplication, or TNT Duping, was first discovered in 1.7 but was not used as it was slightly impractical before the addition of slime blocks. It gained popularity for the first time when upcoming technical Minecraft player ilmango released a video on YouTube entitled "The BEST Way To Make Perimeters in Minecraft," showcasing a slimestone flying machine that used flying TNT Dupers and liquid clearing machines to create a large hole fully automatically.
+TNT Duplication, or TNT Duping, was first discovered in 1.7 but was not used as it was slightly impractical before the addition of slime blocks.
+It gained popularity with the invention of world eaters.
+### Opposition
+Mojang historically opposes TNT Duping as they consider it an exploit, and tried to fix it, but then have decided to leave it in the game for the time being, due to a lack of replacement method
```

Comment:
they tried to fix it 4 times should be mentioned, but in 1.16 they decided to keep it until better methods was implemented